### DISCUSSION TITLE
How to use mitmproxy as just a proxy server and without MITM

### DISCUSSION THREAD

**Discussion by lord8266** [mitmproxy/mitmproxy#6056] [Q&A]
Is there a way to not change the tls certificate ie use this as a proxy server without mitm. I know I can search for some other proxy server but I was wondering if I can use this itself?

**Comment by lord8266** [comment]
Anyone?

**Reply by jonfedric** [reply]
Bro, can i contact you?

**Comment by nneonneo** [comment]
Yes, you can set `ignore_hosts` to `.*` to ignore (pass-through) all SSL traffic, e.g. via the `--set ignore_hosts='.*'` CLI argument. Note that mitmproxy will still intercept any HTTP traffic, although that should not be observable to the client.
